components/eventing-controller/controllers/subscription/jetstream/reconciler_test.go

```diff
@@ -495,8 +495,41 @@ func TestChangeSubscription(t *testing.T) {
 	t.Cleanup(ens.Cancel)
 }
 
-// TODO: TestEmptyEventTypePrefix to be implemented once
-// https://github.com/kyma-project/kyma/issues/13544 is resolved
+// TestEmptyEventTypePrefix tests if a subscription is reconciled properly if the EventTypePrefix is empty.
+func TestEmptyEventTypePrefix(t *testing.T) {
+	ctx := context.Background()
+	g := gomega.NewGomegaWithT(t)
```

Comment:
Why do we use `gomega` here?

Reply:
The reconciler tests are still using gomega, we will get rid of gomega in a separate ticket.